### PR TITLE
FOGL-2772 Centralised DEB packaging fixes

### PR DESCRIPTION
**Pull request by ashish-jabble**
**Fixed items:**
1) FOGL-2840
2) make_deb removed
3) arch fixes on Raspberry target **armv7l**

### CODE REVIEW

File: Package [hunk 1]
```diff
@@ -8,25 +8,24 @@ plugin_install_dirname=${plugin_name}
...
-	aarch64)
-		;;
+    x86_64)
+        echo "${plugin_name} is not supported on Intel architectures as it requires ARM GPIO support"
+        exit 1
+        ;;
+    armv7l)
+        requirements="${requirements},wiringpi"
+        ;;
+    aarch64)
```
Comment (ashish-jabble) [inline Package:21]: @MarkRiddoch Kindly advise on requirements for `aarch64` seems like _wiringpi_ package is not available.

Comment (MarkRiddoch) [reply Package:21]: Yes, the same as a RPi

**Review by MarkRiddoch — APPROVED** [review]

File: Package (hunk 1 above)
Comment (ashish-jabble) [reply Package:21]: `wiringpi` package is not available on `aarch64`. We need to find out the way to install on coral board.

Comment (MarkRiddoch) [reply Package:21]: OK, this is not a priority as it is doubtful anybody is gong to be using a Coral board for this. Let's keep it as a background thing.

**Pull request merged**